### PR TITLE
add help button to tags config panel

### PR DESCRIPTION
**Pull request by Villhellm**
## Proposed change

Added help button + dialog to tags config panel

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/tags/ha-config-tags.ts [hunk 1]
```diff
@@ -193,6 +195,9 @@ export class HaConfigTags extends SubscribeMixin(LitElement) {
         .noDataText=${this.hass.localize("ui.panel.config.tags.no_tags")}
         hasFab
       >
+        <mwc-icon-button slot="toolbar-icon" @click=${this._showHelp}>
+          <ha-svg-icon .path=${mdiHelpCircle}></ha-svg-icon>
```
Comment (zsarnett) [inline src/panels/config/tags/ha-config-tags.ts:199]: is this imported?

**Review by zsarnett — APPROVED** [review]

File: src/panels/config/tags/ha-config-tags.ts [hunk 2]
```diff
@@ -204,6 +209,35 @@ export class HaConfigTags extends SubscribeMixin(LitElement) {
...
+          "companion_link",
+          html`<a
+            href="https://companion.home-assistant.io/"
+            target="_blank"
+            rel="noreferrer"
+            >${this.hass!.localize(
+              "ui.panel.config.tags.detail.companion_apps"
+            )}</a
+          >`
+        )}
```
Comment (bramkragten) [inline src/panels/config/tags/ha-config-tags.ts:227]:
```suggestion
        <p>
        ${this.hass.localize(
          "ui.panel.config.tags.detail.usage",
          "companion_link",
          html`<a
            href="https://companion.home-assistant.io/"
            target="_blank"
            rel="noreferrer"
            >${this.hass!.localize(
              "ui.panel.config.tags.detail.companion_apps"
            )}</a
          >`
        )}
        </p>
```

File: src/panels/config/tags/ha-config-tags.ts (hunk 1 above)
Comment (Villhellm) [reply src/panels/config/tags/ha-config-tags.ts:199]: As far as I can tell all the necessary imports are there. I added mwc-icon-button, but it doesn't look like ha-svg-icon import is necessary for some reason. I'm not super familiar with this system yet though

Comment (bramkragten) [reply src/panels/config/tags/ha-config-tags.ts:199]: it is needed, but probably already is imported by a parent. You should add it though if the order ever changes it breaks.

**Comment by zsarnett** [comment]
RIP. ha-svg-icon was never imported

**Pull request merged**